conans/test/unittests/model/editable_cpp_info/parse_test.py

```diff
@@ -5,96 +5,41 @@
-        self.assertFalse(editable_cpp_info._uses_namespace)
-
-        data = editable_cpp_info._data
-        self.assertListEqual(sorted(data.keys()), sorted(EditableCppInfo.cpp_info_dirs))
-        self.assertListEqual(list(data['includedirs']), ['dirs/includedirs', ])
-        self.assertListEqual(list(data['libdirs']), ['dirs/libdirs', ])
-        self.assertListEqual(list(data['resdirs']), ['dirs/resdirs', ])
-        self.assertListEqual(list(data['bindirs']), [])
-
-    def test_namespace(self):
-        save(self.layout_filepath, self.content)
-        editable_cpp_info = EditableCppInfo.load(self.layout_filepath, require_namespace=True)
-        self.assertTrue(editable_cpp_info._uses_namespace)
-
-        self.assertListEqual(sorted(editable_cpp_info._data.keys()), sorted(['*', 'libA', ]))
-
-        # Check '*'
-        data = editable_cpp_info._data['*']
-        self.assertListEqual(sorted(data.keys()),
-                             sorted(EditableCppInfo.cpp_info_dirs))
-        self.assertListEqual(list(data['includedirs']), ['all/dirs/includedirs', ])
-        self.assertListEqual(list(data['libdirs']), ['all/dirs/libdirs', ])
-        self.assertListEqual(list(data['resdirs']), ['all/dirs/resdirs', ])
-        self.assertListEqual(list(data['bindirs']), [])
-
-        # Check 'libA'
-        data = editable_cpp_info._data['libA']
-        self.assertListEqual(sorted(data.keys()),
-                             sorted(EditableCppInfo.cpp_info_dirs))
-        self.assertListEqual(list(data['includedirs']), ['libA/dirs/includedirs', ])
-        self.assertListEqual(list(data['libdirs']), ['libA/dirs/libdirs', ])
-        self.assertListEqual(list(data['resdirs']), ['libA/dirs/resdirs', ])
-        self.assertListEqual(list(data['bindirs']), [])
-
     def tearDown(self):
         shutil.rmtree(self.test_folder)
+
+    def field_error_test(self):
+        content = textwrap.dedent("""
+                            [includedrs]
+                            something
+                            """)
+        save(self.layout_filepath, content)
+        with self.assertRaisesRegexp(ConanException, "Wrong cpp_info field: includedrs"):
+            _ = EditableCppInfo.load(self.layout_filepath, allow_package_name=False)
+        content = textwrap.dedent("""
+                            [*:includedrs]
+                            something
+                            """)
+        save(self.layout_filepath, content)
+        with self.assertRaisesRegexp(ConanException, "Wrong cpp_info field: includedrs"):
+            _ = EditableCppInfo.load(self.layout_filepath, allow_package_name=True)
+
+    def namespace_error_test(self):
+        content = textwrap.dedent("""
+                            [*:includedirs]
+                            something
+                            """)
+        save(self.layout_filepath, content)
+        with self.assertRaisesRegexp(ConanException, "Repository layout file doesn't allow pattern"):
```

Comment:
This is the reader, it knows nothing about where the file is located (inside/outside the repo). This message is assuming some implicit information.

Reply:
Oh, I see. Yes, lets add the path to the offending file.